cache/metadata.go

```diff
@@ -31,6 +31,7 @@ const keyDiffID = "cache.diffID"
 const keyChainID = "cache.chainID"
 const keyBlobChainID = "cache.blobChainID"
 const keyBlob = "cache.blob"
+const keyForceCompressionBlob = "cache.forceCompressionBlob"
```

Comment:
Let's add setter and getter methods in this file too instead of inlining SetString/GetString where this key is used (helps keep the cache metadata code more maintainable)

Reply:
Agree, (but) will remove this cache key once the new workaround finishes.